frontend/src/App.tsx

```diff
@@ -137,6 +163,54 @@ const App = () => {
     )
   }
 
+  useEffect(() => {
+    if (hasLoadedCourses && hasLoadedStudents) {
+      const interval = setInterval(async () => {
+        const newClasses = (
+          await axios.get(`${API_ROOT}${COURSE_API}`)
+        ).data.map(responseCourseToCourse)
+
+        const newStudents = (
+          await axios.get(`${API_ROOT}${STUDENT_API}`)
+        ).data.map(responseStudentToStudent)
```

Comment:
Some error handling would be nice, e.g. if they lose internet connectivity and we're unable to check if there were updates. I think you can use `setNetworkError()` which will make the snackbar for that appear automatically.

Reply:
For terminology purposes it would also be good to call it newCourses instead of newClasses